README.md

```diff
@@ -45,3 +45,10 @@ git config --local core.hooksPath githooks/
 ```
 
 Alternatively add symlinks in your `.git/hooks` directory to any of the githooks we provide.
+
+## Design goals
+
+This library was built with the primary purpose of talking to Bitcoin Core without the need for
+additional dependencies. This means we do not, and likely never will, support async. If you are
+writing an async application you might be interested in this [alternative JSONRPC
+client](https://github.com/thomaseizinger/rust-jsonrpc-client).
```

Comment:
We should definitely be linking to https://github.com/jeremyrubin/rust-bitcoincore-rpc-async/ as well.

Reply:
Interesting, the lib you link depends on https://github.com/jeremyrubin/rust-jsonrpc-async/ which is, as far as I can tell, a fork of this repo _but_ it has no attribution and was not forked using github (i.e., the github rebo does not indicate it is a fork). It also appears that the same was done for the [json](https://github.com/JeremyRubin/rust-bitcoincore-rpc-async/tree/main/json) crate, it seems to be forked from `rust-bitocincore-rpc/json`, again with no attribution and no indication on github that its a fork.

Both this crate and `rust-bitcoincore-rpc` are licensed under creative commons so its strictly correct to do all this. It  smells bad though.

With this in mind, I'm not going to explicitly link to that crate.

Reply:
The commit history is intact though and @jeremyrubin is a known member of the community. I doubt that there were any bad intentions in regards to missing attributions but probably just an oversight.

In any case, I don't think it is fair to just link to my crate and not the others when they are perfectly fine alternatives. So if you choose to not link to them, please also don't link to mine :)

Reply:
Fair points, thanks.

Reply:
Not accurate to say there is no attribution:

1) License is intact and unmodified, along with all license headers
2) Cargo.toml lists @apoelstra as a co-author

The reason for not being a github fork is because github forks are second class citizens, and there is no intention of upstreaming the patched version, since async is a stated non-goal of the original crate.

Reply:
note also that some of these libraries also depend on the sapio-bitcoin libraries, not the rust-bitcoin ones.

The sapio-bitcoin library is intended to eventually be upstreamed though (it's only real changes are, IIRC, adding schemars support to types, but that proved a bit controversial to de-fork).

Reply:
Excuse the misunderstanding, in the Australian vernacular 'no worries' means 'you are alright, I rescind my concerns, we are all good now'.

Not that anyone else should act like me but I tend to go overboard on the attribution, I would have expected to see in the readme something like:

"I hacked this together with code for XYZ basically just to add 'feature', I don't plan on pushing the changes back to them since 'feature' is not wanted"